asset-bundle-converter/Assets/AssetBundleConverter/LODsConverter/Editor/LODClient.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using UnityEngine;
+using UnityEditor;
+using System.IO;
+using System.Collections.Generic;
+using AssetBundleConverter.LODsConverter.Utils;
+using AssetBundleConverter.Wrappers.Interfaces;
+using UnityEngine.Rendering;
+using Object = UnityEngine.Object;
+
+namespace DCL.ABConverter
+{
+    public class LODClient : MonoBehaviour
+    {
+
+        [MenuItem("Assets/Export URL LODs")]
+        public static void ExportURLLODsToAssetBundles()
+        {
+            string[] commandLineArgs = Environment.GetCommandLineArgs();
+            
+            string customOutputDirectory = "";
+            string lodsURL = "https://lods-bucket-ed4300a.s3.amazonaws.com/-17,-21/LOD/Sources/1707776785658/bafkreidnwpjkv3yoxsz6iiqh3fahuec7lfsqtmkyz3yf6dgps454ngldnu_0.fbx;" +
```

Comment:
do we want these URLs to be public?

Reply:
Ideally, no. We tried integrating the amazon api to make them private; but due to time constraints, we dropped it. We are looking forward to make the downloader using the amazon api in the hardening week. 

Then, they will be private